cfe_internal/enterprise/templates/httpd.conf.mustache

```diff
@@ -215,14 +215,14 @@ LogLevel warn
     object-src 'none'; \
     frame-src 'self'; \
     child-src 'self'; \
-    img-src 'self' avatars.githubusercontent.com badges.gitter.im fonts.gstatic.com kiwiirc.com raw.githubusercontent.com; \
+    img-src 'self' avatars.githubusercontent.com badges.gitter.im fonts.gstatic.com kiwiirc.com raw.githubusercontent.com data:; \
```

Comment:
@craig, this `data:` is already changed in master. It's after self before avitars. Maybe rebase?

Reply:
sure, that's fine...